dir.props

```diff
@@ -174,6 +174,8 @@
   <!-- Set the kind of PDB to Portable and turn on SourceLink (fetching source from GitHub) -->
   <PropertyGroup>
     <DebugType Condition="'$(DebugType)' == ''">Portable</DebugType>
+    <!-- Empty DebugType when building for netfx and in windows so that it is set to full or pdbonly later -->
+    <DebugType Condition="('$(_bc_TargetGroup)'=='netfx' OR $(_bc_TargetGroup.StartsWith('net4'))) AND '$(_runtimeOSFamily)' == 'win'"></DebugType>
```

Comment:
why are you using the _bc_TargetGroup properties as opposed to just TargetGroup?

Instead of using runtimeOSFamily please use $(RunningOnUnix) property

Reply:
because TargetGroup could be netstandard if in the configurations.props netfx is not present right? Or at this level in the execution it hasn't been changed to the bestconfiguration yet?

Reply:
Actually you should use the TargetsNetFx property defined a few lines up for this condition.

Reply:
I think the bc configuration is likely set at this point but we should try to avoid it as if you can as it will be different in a "Build All Configurations" mode. The TargetGroup should be what is used in most cases.